packages/@aws-cdk/region-info/build-tools/generate-static-data.ts

```diff
@@ -38,7 +38,7 @@ async function main(): Promise<void> {
     'us-west-1',
     'us-west-2',
     'ap-south-1',
-    // 'ap-east-1',
```

Comment:
Please leave those commented out - we don't yet have the CDK Metadata resource in those regions.

Reply:
Sure. I comment it out.